bhagavan/mypython/factorial.py

```diff
@@ -0,0 +1,11 @@
+def factorial(n):
+    print("n :", n)
+    
+    f = 1
+    for i in range(1, n+1):
+        f = f * i
+    return f
+
+n = 5
+t = factorial(n)
+print(t)
```

Comment:
the print used here is not supported by python3

Reply:
ACCEPTED: Changes are applied